<launch>
    <node pkg="pointcloud_utils" type="points_playback.py" name="points_playback"
          required="true" output="screen" cwd="node">
         <param name="folder_name" value="innoviz/new" />
         <param name="file_type"   value="xyz" />
         <param name="file_name"   value="innoviz_points" />
         <param name="file_list_name" value="innoviz_file_list.txt" />
         <param name="frame" 	   value="base" />
         <param name="point_topic" value="/leader_points/truth/innoviz" />
    </node>
</launch> 